### PR TITLE
login crash fix, open source info update, readme change

### PR DESCRIPTION
**Pull request by bykovme**

### CODE REVIEW

**Review by bykovme — COMMENTED** [review]
ok

File: src/NSWallet/NSWallet.NetStandard/Views/Pages.cs [hunk 1]
```diff
@@ -330,21 +330,33 @@ public static void ClosePage(INavigation navigation)
...
 
-		public static void Close(bool isModal = false)
+		public static void Unlocker(bool isModal = false)
 		{
-            var navigationPage = Application.Current.MainPage.Navigation;
-
-            if (!isModal && navigationPage.NavigationStack.Count > 0) {
-                Application.Current.MainPage.Navigation.PopAsync(false);
+            // Unhide screen content for iOS devices
+            if (Device.RuntimePlatform == Device.iOS) {
```
Comment (ilyalehchylin) [inline src/NSWallet/NSWallet.NetStandard/Views/Pages.cs:348]: Check this on `NSWallet.cs`

File: src/NSWallet/NSWallet.NetStandard/Views/Pages.cs [hunk 2]
```diff
@@ -330,21 +330,33 @@ public static void ClosePage(INavigation navigation)
 
 		public static void Locker()
 		{
-            Application.Current.MainPage.Navigation.PushModalAsync(
-				new NavigationPage(new LockerPageView()), false);
+			// Hide screen content for iOS devices
+            if (Device.RuntimePlatform == Device.iOS) {
```
Comment (ilyalehchylin) [inline src/NSWallet/NSWallet.NetStandard/Views/Pages.cs:335]: Check this on `NSWallet.cs`

**Review by ilyalehchylin — CHANGES_REQUESTED** [review]
Check single comments

File: src/NSWallet/NSWallet.NetStandard/Views/Pages.cs (hunk 1 above)
Comment (bykovme) [reply src/NSWallet/NSWallet.NetStandard/Views/Pages.cs:348]: Let's leave it there for a while, we need better solution to secure the screen for iOS anyway, we will refactor this later, I will just put a comment there, so we do not forget

File: src/NSWallet/NSWallet.NetStandard/Views/Pages.cs (hunk 2 above)
Comment (bykovme) [reply src/NSWallet/NSWallet.NetStandard/Views/Pages.cs:335]: same as previous

**Review by ilyalehchylin — APPROVED** [review]

**Pull request merged**